cysync/src/utils/networkFees.ts

```diff
@@ -35,7 +35,7 @@ const getFees = async (coinId: string) => {
       );
     }
 
-    const fees = value.dividedBy(1000000000).integerValue().toNumber();
+    const fees = value.dividedBy(1000000000).toNumber();
```

Comment:
Need to round off this value

Reply:
changed. now using precision of 3-digits